tests/integration/tls_tests/test_tls.py

```diff
@@ -78,33 +88,56 @@ async def check_certs_correctly_distributed(ops_test: OpsTest, unit: Unit) -> No
         assert relation_internal_cert == internal_contents_file
 
 
+@pytest.mark.skipif(
+    os.environ.get("PYTEST_SKIP_DEPLOY", False),
+    reason="skipping deploy, model expected to be provided.",
+)
```

Comment:
remove this for the same reasons that we removed the other one

Reply:
done, thank you